sky/global_user_state.py

```diff
@@ -34,7 +34,7 @@
 def create_table(cursor, conn):
     # Enable WAL mode to avoid locking issues.
     # See: issue #1441 and PR #1509
-    conn.execute('PRAGMA journal_mode=WAL')
+    cursor.execute('PRAGMA journal_mode=WAL')
```

Comment:
Reason in PR description seems truncated:
> Also, we changed to using cursor to run the command for setting WAL mode instead of connection as we
As our autostop_lib

Reply:
Oops, just updated.

Reply:
It is to make the usage of `conn` and `cursor` consistent and avoid an unnecessary `commit` the `conn.execute` implies.